.github/workflows/ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - run: |
+          wget https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/logwarn/logwarn-1.0.11.tar.gz
+          tar xvf logwarn-1.0.11.tar.gz
+          cd logwarn-1.0.11
+          ./configure --prefix=/usr && make && sudo make install
```

Comment:
Instead of installing with `sudo` one could also just install it into a writable prefix (and add its bin directory to the path). I suppose that's generally preferable over invoking build system commands as root. (And to it really needed to go under `/usr` the cleanest way would be a staged installation via `DESTDIR`.)

Reply:
I just copied it from `.travis.yml` for now, so I can start with my actual work, but I will try to change it.

Reply:
done